### PR TITLE
Updated readme.md to include faker for Currency Codes.

### PR DESCRIPTION
**Pull request by stelgenhof**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,5 @@
 vendor
 composer.lock
+.idea
```
Comment (fzaninotto) [inline .gitignore]: Please don't add IDE-specific lines to the project's `.gitignore`. These files should be in your own user's `.gitignore`.

File: phpunit.xml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (fzaninotto) [inline phpunit.xml]: Do not commit this.

**Comment by fzaninotto** [comment]
No need of 5 commits for 1 line. I have pushed the change in 81e4eccea302300ac0c69c7ed04c94e29f580633.

**Comment by stelgenhof** [comment]
Ah! My apologies... I didn't mean to include those files into the PR. They were meant for my own fork.

**Comment by TimWolla** [comment]
> Ah! My apologies... I didn't mean to include those files into the PR. They were meant for my own fork.

Always remember to create a new branch for a new pull request :smile:

**Pull request closed without merging**